src/etc/inc/gwlb.inc

```diff
@@ -864,6 +869,15 @@ function return_gateways_array($disabled = false, $localhost = false, $inactive
 		$gateways_arr['Null4'] = $gwlo4;
 		$gateways_arr['Null6'] = $gwlo6;
 	}
+
+	if ($integer_index) {
+		$gateways_arr_temp = array();
```

Comment:
This whole block could be reduced to: `$gateways_arr = array_values($gateways_arr);`

Or maybe just use that where it's needed rather than adding it to the function?